pom.xml

```diff
@@ -2121,12 +2121,100 @@
                     <version>1.2.8</version>
                     <configuration>
                         <dynamicDependencies>
+                            <DynamicDependency>
+                                <groupId>org.apache.maven.plugins</groupId>
+                                <artifactId>maven-javadoc-plugin</artifactId>
+                                <version>3.2.0</version>
+                                <repositoryType>PLUGIN</repositoryType>
+                            </DynamicDependency>
+                            <DynamicDependency>
+                                <groupId>org.apache.yetus</groupId>
+                                <artifactId>audience-annotations</artifactId>
```

Comment:
`Add dynamic dependencies found by CI` should probably come before `Add Maven --offline flag as a default option`

i think this should be part of `Add different plugin to download more dependencies` commit
and that commit title could be changed to `Download dynamic build dependencies with retries`